### PR TITLE
Headline and other stylings for blogs before August 2014

### PR DESCRIPTION
**Pull request by emileighoutlaw**
Capitalization for blogs up to August 2014. (small piece of effort to align our early work with our current content guide).

### CODE REVIEW

File: _posts/2014-03-12-coming-soon.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 - 18f
 
 ---
-<p><span>We&#8217;re truly gratified so many of you liked our sneak preview! We&#8217;ll be back real soon. </span></p>
+<p><span>Were truly gratified so many of you liked our sneak preview! We'll be back real soon. </span></p>
```
Comment (gboone) [inline _posts/2014-03-12-coming-soon.html]: Don't we still need the apostrophe on "we're?"

File: _posts/2014-03-21-29-minutes.html [hunk 2]
```diff
@@ -15,11 +15,11 @@
 ---
 
 <p>I want to relate a great little thing that happened here at <a href="http://gsa.gov/">18th and F</a> yesterday.</p>
-<p>At 11am yesterday morning, members of the GSA.gov team, Sarah Hyder and Patrick Son, met with Hillary Hartley, Alison Rowland, and me to talk about how to integrate <a href="https://fbopen.gsa.gov">FBOpen</a> into GSA&#8217;s mobile website, <a href="http://m.gsa.gov">m.gsa.gov</a>, on its &#8220;<a href="http://m.gsa.gov/m/#!/buy-sell/how-to-sell">How To Sell to the Government</a>" page.</p>
+<p>At 11am yesterday morning, members of the GSA.gov team, Sarah Hyder and Patrick Son, met with Hillary Hartley, Alison Rowland, and me to talk about how to integrate <a href="https://fbopen.gsa.gov">FBOpen</a> into GSAs mobile website, <a href="http://m.gsa.gov">m.gsa.gov</a>, on its <a href="http://m.gsa.gov/m/#!/buy-sell/how-to-sell">How To Sell to the Government</a>" page.</p>
```
Comment (gboone) [inline _posts/2014-03-21-29-minutes.html]: Still need the apostrophe here, too. `&#8217;` is the encoding for an apostrophe but I think a straight `'` will do just fine. Worth a shot anyway.

File: _posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.html [hunk 3]
```diff
@@ -3,7 +3,7 @@
 date: '2014-05-14T11:02:00-04:00'
 tumblr_url: http://18fblog.tumblr.com/post/85724876053/hacking-bureaucracy-improving-hiring-and-software
 
-title: 'Hacking Bureaucracy: improving hiring and software deployment'
+title: "Hacking bureaucracy: improving hiring and software deployment"
```
Comment (gboone) [inline _posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.html]: Should "improving" be capitalized?

File: _posts/2014-07-16-midas-a-marketplace-for-innovation-in-government.html [hunk 4]
```diff
@@ -3,7 +3,7 @@
 date: '2014-07-16T11:01:00-04:00'
 tumblr_url: http://18fblog.tumblr.com/post/91952641466/midas-a-marketplace-for-innovation-in-government
 
-title: "Midas: A Marketplace for Innovation in Government"
+title: "Midas: a marketplace for innovation in government"
```
Comment (gboone) [inline _posts/2014-07-16-midas-a-marketplace-for-innovation-in-government.html]: Should the "a" be capitalized?

File: _posts/2014-07-29-18f-an-open-source-team.html [hunk 5]
```diff
@@ -3,7 +3,7 @@
 date: '2014-07-29T14:01:55-04:00'
 tumblr_url: http://18fblog.tumblr.com/post/93226835135/18f-an-open-source-team
 
-title: '18F: An Open Source Team'
+title: "18F: an open source team"
```
Comment (gboone) [inline _posts/2014-07-29-18f-an-open-source-team.html]: Should "an" be capitalized?

File: _posts/2014-03-21-29-minutes.html (hunk 2 above)
Comment (gboone) [inline _posts/2014-03-21-29-minutes.html]: Scrolling down it looks like you already knew that!

File: _posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.html (hunk 3 above)
Comment (emileighoutlaw) [inline _posts/2014-05-14-hacking-bureaucracy-improving-hiring-and-software.html]: Only if it's a complete sentence after the colon. So "Hacking bureaucracy: We're improving hiring and software deployment" would need the caps.

File: _posts/2014-07-16-midas-a-marketplace-for-innovation-in-government.html (hunk 4 above)
Comment (emileighoutlaw) [inline _posts/2014-07-16-midas-a-marketplace-for-innovation-in-government.html]: Same as above. :)

File: _posts/2014-03-21-29-minutes.html (hunk 2 above)
Comment (emileighoutlaw) [inline _posts/2014-03-21-29-minutes.html]: Ah! I did *not* know that was the encoding for an apostrophe. I kept pulling down from the site and couldn't figure out why it was replacing all my punctuation with a jumble of symbols! Newbie mistake, haha.

File: _posts/2014-03-12-coming-soon.html (hunk 1 above)
Comment (emileighoutlaw) [inline _posts/2014-03-12-coming-soon.html]: Yep! Fixing now!

**Comment by gboone** [comment]
:+1: :+1: :+1:

**Pull request merged**